generators/firechip/src/main/scala/BridgeBinders.scala

```diff
@@ -98,7 +100,59 @@ class WithUARTBridge extends OverrideHarnessBinder({
 class WithBlockDeviceBridge extends OverrideHarnessBinder({
   (system: CanHavePeripheryBlockDevice, th: FireSim, ports: Seq[ClockedIO[BlockDeviceIO]]) => {
     implicit val p: Parameters = GetSystemParameters(system)
-    ports.map { b => BlockDevBridge(b.clock, b.bits, th.harnessReset.toBool) }
+    ports.map { b => BlockDevBridge(b.clock, b.bits, th.harnessReset.asBool) }
+    Nil
```

Comment:
What was the effect of leaving this off? Was this code just not exercised before?

Reply:
I'm not sure I follow. I think the only thing that changed here was that I converted `toBool` to `asBool` (the diff makes this a bit hard to see). If you are referring to removing the block device... there is no block device on the test chip so I removed it (forcing people to use the initramfs linux version).

Reply:
I was referring to adding `Nil`.

Reply:
`Nil` was already added before. The diff makes this hard to see. https://github.com/ucb-bar/chipyard/pull/812/files#diff-1cd9711add00014e504e37e13d8aa69c28654c356c53e6b84bddd104c11a4763L102

Reply:
oh wow. that is hard to see. Thanks